pubtools/_quay/signature_handler.py

```diff
@@ -129,7 +129,7 @@ def create_manifest_claim_message(
         }
         return message
 
-    def get_tagged_image_digests(self, image_ref):
+    def get_tagged_image_digests(self, image_ref, media_type):
```

Comment:
So, this still doesn't work for v2s1? Do you intend to merge broken code?

Reply:
Right, I forgot to apply fixes for this part. Should be fixed in the latest commit